test/mlir/driver/product.version.mlir

```diff
@@ -0,0 +1,16 @@
+// RUN: onnx-mlir --preserveBitcode %s -o %t
+// RUN: llvm-dis %t.bc -o %t.ll
+// RUN: cat %t.ll | FileCheck %s
+// XFAIL: onnx-mlir-product-version
```

Comment:
I think this test should pass always. We don't need to verify that the product version is 0, but that it exists. Then you can skip both this XFAIL as well as the changes to the lit cfg files

Reply:
Good idea, changed.